react/week2/interactive-todo-list/src/components/CreateTodo.jsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+// import { ReactDOM } from "react-dom";
+import { useState } from "react";
+import TodoList from "./TodoList";
+import swal from "sweetalert";
+import Counter from "./Counter";
+
+export default function CreateTodo() {
+  const [todo, setTodo] = useState({ title: "", done: false });
+  const [todoArr, setTodoArr] = useState([]);
+
+  const handleChange = (event) => {
+    let { value } = event.target;
+    let obj = {};
+    obj["title"] = value;
+    obj["done"] = false;
+    setTodo(obj);
```

Comment:
The problem is because you use indexing to identify the todo which is changing when the list is updated. 
Therefor we have to add `id` to each todo
```suggestion
    let obj = {};
    obj["title"] = value;
    obj["done"] = false;
    obj["id"]= Date.now()
    setTodo(obj);
```

the `Date.now()` is not best practice to get a unique id, but it's a quick fix for now

Reply:
ok , thanks a lot for the suggestion, so we need to avoid using index for identifying the todo.